services/ui-src/src/components/fields/TextField.tsx

```diff
@@ -27,14 +20,6 @@ export const TextField = ({
 }: Props) => {
   const defaultValue = "";
   const [displayValue, setDisplayValue] = useState<string>(defaultValue);
-
-  /*
-   * Last value is the last value that was submitted to Autosave. We only want to submit
-   * To autosave if theres been a change
-   */
-  const [lastAutosaveValue, setLastAutosaveValue] =
-    useState<string>(defaultValue);
```

Comment:
Note: I don't think we need this anymore because we can rely on the `hydrationData` updating from the database on every `updateReport` call as I noted in the other PR's comments, but I might be missing something, so let's chat about it.